src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  // box-sizing: border-box;
+  border: 1px solid $light-color-border;
+  border-radius: 5px;
+  background-color: $white-color-link;
+  padding: 16px;
```

Comment:
You can write like so: padding: `32px 16px 16px ` and remove padding top property